src/main/search/handleSearch.java

```diff
@@ -0,0 +1,80 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
```

Comment:
These packages do not exist. Make sure to update `build.gradle` appropriately to include the necessary dependencies.

Reply:
Oops, added to build file!